modles/postModel.js

```diff
@@ -3,26 +3,37 @@ function postArticleMdl(req) {
   const { article_id } = req.params;
   return db
     .query(
-      "INSERT INTO comments (body, author, article_id, votes)VALUES($1,$2,$3,$4) RETURNING *;",
-      [req.body[0]["body"], req.body[0]["username"], article_id, 0]
+      "INSERT INTO comments (body, author, article_id, votes,created_at)VALUES($1,$2,$3,$4,$5) RETURNING *;",
+      [
+        req.body[0]["body"],
+        req.body[0]["username"],
+        article_id,
+        0,
+        "2023-01-11T00:00:00.000Z",
```

Comment:
we don't need to be passing in the votes and created_at here as all we require is the body, username and article_id